package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "gp-v2-trading-bot",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint --max-warnings 0 ."
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
```

Comment:
ubernit: I think some of these should eventually be `dependencies` as you will probably need to `import` it in order to run. (things like TSC and linting should still be `devDependencies` though).